x/mint/keeper/keeper.go

```diff
@@ -262,11 +265,11 @@ func (k Keeper) distributeDeveloperRewards(ctx sdk.Context, totalMintedCoin sdk.
 	} else {
 		// allocate developer rewards to addresses by weight
 		for _, w := range developerRewardsReceivers {
-			devPortionCoin, err := getProportions(devRewardCoin, w.Weight)
+			devPortionAmount, err := getProportion(developerRewardsCoin.Amount.ToDec(), w.Weight)
```

Comment:
Note to reviewers: Currently, we provide the truncated value from `developerRewardsCoin.Amount` here for state compatibility. However, later we will change this to:
```
devPortionAmount, err := getProportion(devRewardsAmount, w.Weight)
```

This change helps to reduce the truncation delta from 2600 to 300 OSMO in the first year according to tests